spec/system/support_interface/authentication_spec.rb

```diff
@@ -24,7 +26,14 @@ def when_i_visit_the_support_interface
     visit support_interface_applications_path
   end
 
-  def and_i_sign_in_via_dfe_sign_in
+  def then_i_should_be_redirected_to_login_page
+    expect(page).to have_current_path support_interface_sign_in_path
+    expect(page).not_to have_link 'Applications'
```

Comment:
It's surely right to hide this stuff on the sign-in page but maybe it's a separate assertion in this spec? Or maybe it doesn't need testing here?

Reply:
Have created a separate `and_i_should_not_see_support_menu` method for these assertions.